.github/workflows/linter.yml

```diff
@@ -0,0 +1,51 @@
+---
+###########################
+###########################
+## Linter GitHub Actions ##
+###########################
+###########################
+name: Lint Code Base
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+
+#############################
+# Start the job on all push #
+#############################
+on:
+  push:
+    branches-ignore:
+      - main
+
+###############
+# Set the Job #
+###############
+jobs:
+  build:
```

Comment:
`lint` or `analyze` would be better here. There's nothing being built in this workflow.

Reply:
I vote to keep the "defaults" mentioned here: https://github.com/github/super-linter#example-connecting-github-action-workflow 

Let me know if you want to really change it to `lint` ...